src/main/java/net/pms/newgui/TranscodingTab.java

```diff
@@ -675,16 +681,54 @@ public void itemStateChanged(ItemEvent e) {
 		});
 		builder.add(GuiUtil.getPreferredSizeComponent(channels), FormLayoutUtil.flip(cc.xy(3, 2), colSpec, orientation));
 
+
+		builder.addLabel(Messages.getString("TrTab2.29"), FormLayoutUtil.flip(cc.xy(1, 4), colSpec, orientation));
+		abitrate = new JTextField("" + configuration.getAudioBitrate());
+		abitrate.addKeyListener(new KeyAdapter() {
+			@Override
+			public void keyReleased(KeyEvent e) {
+				try {
+					int ab = Integer.parseInt(abitrate.getText());
+					configuration.setAudioBitrate(ab);
+				} catch (NumberFormatException nfe) {
+					LOGGER.debug("Could not parse audio bitrate from \"" + abitrate.getText() + "\"");
+				}
+			}
+		});
+		builder.add(abitrate, FormLayoutUtil.flip(cc.xy(3, 4), colSpec, orientation));
+
+		builder.addLabel(Messages.getString("MEncoderVideo.7"), FormLayoutUtil.flip(cc.xy(1, 6), colSpec, orientation));
+		langs = new JTextField(configuration.getAudioLanguages());
+		langs.setToolTipText(Messages.getString("TrTab2.75"));
+		langs.addKeyListener(new KeyAdapter() {
+			@Override
+			public void keyReleased(KeyEvent e) {
+				configuration.setAudioLanguages(langs.getText());
+			}
+		});
+		builder.add(langs, FormLayoutUtil.flip(cc.xy(3, 6), colSpec, orientation));
+
 		forcePCM = new JCheckBox(Messages.getString("TrTab2.27"), configuration.isAudioUsePCM());
 		forcePCM.setToolTipText(Messages.getString("TrTab2.83"));
 		forcePCM.setContentAreaFilled(false);
 		forcePCM.addItemListener(new ItemListener() {
 			@Override
 			public void itemStateChanged(ItemEvent e) {
 				configuration.setAudioUsePCM(e.getStateChange() == ItemEvent.SELECTED);
+				AudioRemuxLPCM.setEnabled((e.getStateChange() != ItemEvent.SELECTED));
 			}
 		});
-		builder.add(GuiUtil.getPreferredSizeComponent(forcePCM), FormLayoutUtil.flip(cc.xy(1, 4), colSpec, orientation));
+		builder.add(GuiUtil.getPreferredSizeComponent(forcePCM), FormLayoutUtil.flip(cc.xy(1, 8), colSpec, orientation));
+
+		AudioRemuxLPCM = new JCheckBox(Messages.getString("TrTab2.30"), configuration.isAudioRemuxLPCM());
+		AudioRemuxLPCM.setToolTipText(Messages.getString("TrTab2.84") + (Platform.isWindows() ? " " + Messages.getString("TrTab2.21") : "") + "</html>");
```

Comment:
Using `TrTab2.84` is wrong. Default value for `audio_remux_lpcm =` is `false` but the `TrTab2.84` says `Default: Enabled`. This must be corrected also for other sethings.